cpp/cmake/modules/FindCSP.cmake

```diff
@@ -29,75 +29,114 @@
 
 cmake_minimum_required(VERSION 3.7.2)
 
-find_package(Python ${CSP_PYTHON_VERSION} EXACT REQUIRED COMPONENTS Interpreter)
 
-set(ENV{PYTHONPATH} "${CMAKE_SOURCE_DIR}/ext:$ENV{PYTHONPATH}")
-
-# Find out the base path
-execute_process(
-  COMMAND "${Python_EXECUTABLE}" -c
-          "from __future__ import print_function;import os.path;import csp;print(os.path.dirname(csp.__file__), end='')"
-          OUTPUT_VARIABLE __csp_base_path)
+if(EXISTS "${CMAKE_SOURCE_DIR}/csp/__init__.py")
+  set(CSP_IN_SOURCE_BUILD ON)
+  set(__csp_base_path "${CMAKE_SOURCE_DIR}/csp")
+  set(__csp_include_path "${CMAKE_SOURCE_DIR}/cpp")
+  set(__csp_lib_path "${CMAKE_SOURCE_DIR}/")
+  set(__csp_base_path "${CMAKE_SOURCE_DIR}/csp")
```

Comment:
duplicate with line 35 and the next line, i mustve had a stroke while writing this